app/src/main/java/ch/epfl/qedit/model/Question.java

```diff
@@ -14,20 +13,30 @@
 
     /**
      * Treasure hunt questions also have a location and a radius within which they can be answered.
+     * android.Location is not serializable, so we store the longitude and latitude instead.
```

Comment:
Yeah, I had to do that move, because I couldn't pass the quiz in the bundle because of that. It sucks that the android API is split between `Parcelable` and `Serializable`. I wish they could have a unified API, but again I was disappointed. They prefer having two APIs and deprecating one, it seems ;).